tensorflow/core/lib/jpeg/jpeg_mem.cc

```diff
@@ -593,6 +593,11 @@ bool GetImageInfo(const void* srcdata, int datasize, int* width, int* height,
 namespace {
 bool CompressInternal(const uint8* srcdata, int width, int height,
                       const CompressFlags& flags, tstring* output) {
+  if (output == nullptr)
+    LOG(ERROR) << "Output buffer is null: ";
+    return false;
```

Comment:
Hi @gaurav1086 If we have find the NULL point, will should we just return false? How about create a new tstring object?

Reply:
@Leslie-Fang , thanks a lot for the review. I appreciate your pointer, however I am under the impression (have to double check) that the caller of this function (GetImageInfo) should/would check the return value (bool: true/false), before trying to access the output buffer. Thoughts ? This function returns false at other occasions, hence I thought of returning  a false here as well.